1.8/artifactory/artifactory-enterprise.md

```diff
@@ -14,14 +14,20 @@
 
 ## Setting up Artifactory Enterprise
 
-1. Mount NFS Storage to Each Private Node of DC/OS Cluster.<br />
-    For example: You have artifactoryha.mount.com:/artifactory as your mount point.
-    Mount it to /var/data/artha dir of your each private node of DC/OS.
-      ```sudo mount artifactoryha.mount.com:/artifactory /var/artifactory/```
-    All the nodes should share the same file storage, for now the only way is to use NFS. This requirement will be removed in the future.<br />
-    Note: Provide permission to write and create subdirectories to /var/data/artha.
+### 1. Mount NFS Storage to Each Private Node of DC/OS Cluster
+All nodes should share the same file storage. For now, the only supported method is with NFS, but this requirement will be removed in the future.
 
-2. Create a new file called `artifactory-enterprise-options.json`, with the following content (replace the license parameter with your own license string):
+As an example, if `artifactoryha.mount.com:/artifactory` is the mount point, mount it to `/var/data/artha` on each private node of DC/OS with the following command:
+
+```
+sudo mount artifactoryha.mount.com:/artifactory /var/artifactory/
```

Comment:
Pretty sure `/var/artifactory` is incorrect and should be `/var/data/artha` as mentioned in the previous sentence.